src/components/DayPicker.jsx

```diff
@@ -751,9 +750,7 @@ export default class DayPicker extends React.Component {
 
     const horizontalWidth = (calendarMonthWidth * numberOfMonths) + (2 * DAY_PICKER_PADDING);
 
-    // this is a kind of made-up value that generally looks good. we'll
-    // probably want to let the user set this explicitly.
-    const verticalHeight = 1.75 * calendarMonthWidth;
+    const verticalHeight = (calendarMonthWidth * numberOfMonths) + (2 * MONTH_PADDING);
```

Comment:
We should still probably leave a comment as to the effect of this number, but yes, I agree this is better.